terraform/projects/app-ecs-albs/main.tf

```diff
@@ -176,6 +204,19 @@ resource "aws_lb_listener" "nginx_auth_external_listener" {
   }
 }
 
+resource "aws_lb_listener" "nginx_auth_external_listener_https" {
+  load_balancer_arn = "${aws_lb.nginx_auth_external_alb.arn}"
+  port              = "443"
+  protocol          = "HTTPS"
+  ssl_policy        = "ELBSecurityPolicy-2016-08"
+  certificate_arn   = "${aws_acm_certificate.monitoring_cert.arn}"
+
+  default_action {
+    target_group_arn = "${aws_lb_target_group.nginx_auth_external_endpoint.0.arn}"
+    type             = "forward"
+  }
+}
+
 resource "aws_lb_listener_rule" "prom_public_listener" {
```

Comment:
I believe so as per https://stackoverflow.com/questions/47032187/terraform-aws-lb-listener-with-multiple-ports

Reply:
I don't think I made myself particularly clear. What I'm trying to say is: do we need to duplicate the `aws_lb_listener_rule` resources (which hasn't been done), *as well as* the `aws_lb_listener` resource (which has already been done above)

Reply:
My thinking is that: because the `aws_lb_listener_rule`s refer to the HTTP `aws_lb_listener`, I wondered if we needed similar rules for the HTTPS listener too.

Reply:
Ah I understand you now. Good question. I need to bit of reading to make sure I understand how they all work together so let me get back to you on this one...